lib/services/local/configuration.go

```diff
@@ -401,5 +430,5 @@ const (
 	auditPrefix            = "audit"
 	networkingPrefix       = "networking"
 	sessionRecordingPrefix = "session_recording"
-	installerScriptPrefix  = "installer_script"
+	installerScriptPrefix  = "installer_scripts"
```

Comment:
Since we're going to be adding other script types soon, how about we store it under `scripts/installer` in the backend? So let's make `scriptsPrefix = "scripts"` and `installerPrefix = "installer"` and assemble the backend key from them.

Reply:
Yes.